docs/packages_wiki/atom-languageclient/release-process.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Release Process
+---
+
+# Release Process 
+
+::: warning
```

Comment:
Just as a remark: GitHub has introduced its own syntax for warnings. We should possibly keep an eye on this to see if VuePress (or VitePress) has plans to switch to the same syntax. I should mention though that GitHub's approach has raised some eyebrows since it's using bad semantics by simply highjacking the `<blockquote>` element for its own purposes, so personally, I wouldn't recommend using GitHub's version.

Reply:
Thats good to know. We can keep an eye out to see if we can have our Markdown look good on both GitHub and Vuepress, but for now sounds like we should just stick with the current syntax.